src/sentry/replays/usecases/ingest/dom_index.py

```diff
@@ -396,6 +396,15 @@ def _handle_breadcrumb(
 ) -> ReplayActionsEventPayloadClick | None:
 
     click = None
+    project = Project.objects.get(id=project_id)
+    dead_click_timeout = (
+        5000
+        if features.has(
+            "organizations:session-replay-dead-click-reduced-timeout",
+            project.organization,
+        )
+        else 7000
```

Comment:
Ideally the option would hold the value instead. So you don't need to branch based on a bool flag and have the values hard coded here.

Will make it easier to tweak the value later (say we go from 5 to 3 seconds for example)

Reply:
You need two options because flag pole only returns booleans.